pom.xml

```diff
@@ -46,7 +58,33 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
+    
     <dependencies>
+      <dependency>
+        <groupId>org.jenkins-ci.plugins.workflow</groupId>
+        <artifactId>workflow-aggregator</artifactId>

```

Comment:
Do not use this plugin as a dependency even for tests. Specify `workflow-cps`, `workflow-job`, and any step plugins you need.